lib/settings.js

```diff
@@ -122,10 +122,10 @@ class Settings {
     const template = eta.loadFile(path.join(__dirname, 'summary.tmpl'))
     const commentmessage = await eta.render(template, stats)
 
-    /*
-    const commentmessage2 = `
-## :robot: Safe-Settings Validator summary
-### Run on: ${new Date()}
```

Comment:
I removed this since the existing GitHub comment already contains when it was created